test/cases/010_load/030_unload/test.ebpf.rs

```diff
@@ -0,0 +1,30 @@
+//! ```cargo
+//! [dependencies]
+//! aya-bpf = { path = "../../../../bpf/aya-bpf" }
+//! ```
+
+#![no_std]
+#![no_main]
+
+use aya_bpf::{
+    bindings::xdp_action,
+    macros::xdp,
+    programs::XdpContext,
+};
+
+#[xdp(name="ihaveaverylongname")]
```

Comment:
this looks like copy paste from some other test. I think it'd  be better to give it a more descriptive name eg "test_unload" or something